test/t8_gtest_macros.hxx

```diff
@@ -46,6 +52,12 @@ auto print_eclass = [] (const testing::TestParamInfo<t8_eclass> &info) { return
 #define T8_NUM_SAMPLE_POINTS 10000
 #endif
 
+const t8_scheme_cxx *default_scheme = t8_scheme_new_default_cxx ();
```

Comment:
Please encapsulate these inside of a namespace to prevent interference with other schemes in the tests.